CHANGELOG.md

```diff
@@ -3,7 +3,94 @@
+- docs: Revise docs index page wording ([#7805](https://github.com/open-policy-agent/opa/pull/7805)) authored by @charlieegan3
+- docs: Update warning note in GraphQL API docs ([#7737](https://github.com/open-policy-agent/opa/pull/7737)) authored by @charlieegan3
+- website: Add wildcard CORS for data/versions.json ([#7784](https://github.com/open-policy-agent/opa/pull/7784)) authored by @charlieegan3
+- website: Ensure no hscroll on built-in tables ([#7773](https://github.com/open-policy-agent/opa/pull/7773)) authored by @charlieegan3
+- website: Render versions under `/data/versions.json` ([#7783](https://github.com/open-policy-agent/opa/pull/7783)) authored by @charlieegan3
+- website: Set mobile and desktop tab sizes ([#7774](https://github.com/open-policy-agent/opa/pull/7774)) authored by @charlieegan3
+- website: Show link to the edge release of the docs ([#7776](https://github.com/open-policy-agent/opa/pull/7776)) authored by @charlieegan3
+
+### Miscellaneous
+
+- Benchmark fixes ([#7765](https://github.com/open-policy-agent/opa/pull/7765)) authored by @anderseknert
+- Use Regal for linting Rego ([#7752](https://github.com/open-policy-agent/opa/pull/7752)) authored by @anderseknert
+- Use shorthand form for types ([#7757](https://github.com/open-policy-agent/opa/pull/7757)) authored by @anderseknert
+- .github: Use types for issues ([#7751](https://github.com/open-policy-agent/opa/pull/7751)) authored by @charlieegan3
+- build: Add top-level token permissions for workflows ([#7795](https://github.com/open-policy-agent/opa/pull/7795)) authored by @timothyklee
+- build: Show a warning when .manifest is ignored ([#7807](https://github.com/open-policy-agent/opa/pull/7807)) authored by @charlieegan3
+- cli: Avoid os.Exit() in Run() funcs ([#7788](https://github.com/open-policy-agent/opa/pull/7788)) authored by @srenatus
+- cmd: Use command.RunE to return errors and perform orderly shutdown of OPA ([#7788](https://github.com/open-policy-agent/opa/pull/7788)) authored by @srenatus
+- docs/build: Link checker fixes ([#7743](https://github.com/open-policy-agent/opa/pull/7743)) authored by @charlieegan3
+- Dependency updates; notably:
+  - build(deps): bump github.com/go-viper/mapstructure/v2 from 2.2.1 to 2.3.0.
+  - build(deps): bump github.com/containerd/containerd/v2 from 2.1.1 to 2.1.3
+  - build(deps): bump github.com/dgraph-io/badger/v4 from 4.7.0 to 4.8.0
+  - build(deps): bump github.com/sergi/go-diff from 1.3.1 to 1.4.0
+  - build(deps): bump github.com/spf13/pflag from 1.0.6 to 1.0.7
+  - build(deps): bump github.com/tchap/go-patricia/v2 from 2.3.2 to 2.3.3
+  - build(deps): bump github.com/vektah/gqlparser/v2 from 2.5.28 to 2.5.30
+  - build(deps): bump golang.org/x/net from 0.41.0 to 0.42.0
+  - build(deps): bump golang.org/x/time from 0.11.0 to 0.12.0
+  - build(deps): bump google.golang.org/grpc from 1.73.0 to 1.74.2
+  - build(deps): bump sigs.k8s.io/yaml from 1.4.0 to 1.6.0
```

Comment:
I think only opentelemetry and gRPC and gqlparser  and _maybe_ containerd are worth taking note of. 🧐